examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>get_metrics from research does exactly that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>